src/sentry_value.c

```diff
@@ -840,6 +827,25 @@ sentry_value_as_string(sentry_value_t value)
     }
 }
 
+sentry_value_t
+sentry__value_ring_buffer_to_list(const sentry_value_t rb)
+{
+    const thing_t *thing = value_as_thing(rb);
+    if (!thing || thing_get_type(thing) != THING_TYPE_LIST) {
+        return sentry_value_new_null();
+    }
+    const list_t *rb_list = thing->payload._ptr;
+    const int32_t start_idx = sentry_value_as_int32(rb_list->items[0]);
+
+    sentry_value_t rv = sentry_value_new_list();
+    for (size_t i = 0; i < rb_list->len - 1; i++) {
+        int32_t idx = (start_idx - 1 + i) % (rb_list->len - 1) + 1;
```

Comment:
Due to API constraints, we must _store_ an `int32` value (or the integral component of a 64-bit float), but we should use it always as an unsigned type here. I think this what the VC compiler complains about.